pkg/proxy/util/nfacct/nfacct_linux.go

```diff
@@ -146,9 +148,17 @@ func (r *runner) Get(name string) (*Counter, error) {
 
 // List is part of the interface.
 func (r *runner) List() ([]*Counter, error) {
+	var err error
+	var msgs [][]byte
+
 	req := r.handler.newRequest(cmdGet, unix.NLM_F_REQUEST|unix.NLM_F_DUMP)
-	msgs, err := req.Execute(unix.NETLINK_NETFILTER, 0)
-	if err != nil {
+	err = retry.OnError(util.MaxAttemptsEINTR, util.ShouldRetryOnEINTR, func() error {
+		msgs, err = req.Execute(unix.NETLINK_NETFILTER, 0)
```

Comment:
Should the `r.handler.newRequest(cmdGet, unix.NLM_F_REQUEST|unix.NLM_F_DUMP)` be included in the retry?
I only moved it out as the current mocking of the tests expects that.